src/crypto/shelley/ShelleyWallet.js

```diff
@@ -507,11 +509,27 @@ export default class ShelleyWallet extends Wallet implements WalletInterface {
       CONFIG.NUMBERS.CHAIN_DERIVATIONS.CHIMERIC_ACCOUNT,
     )).derive(CONFIG.NUMBERS.STAKING_KEY_INDEX)).to_raw_key()
 
+    const wits = new Set()
+
+    const txBuilder: TransactionBuilder = signRequest.unsignedTx
+    // prettier-ignore
+    wits.add(
+      Buffer.from(
+        await (await make_vkey_witness(
+          await hash_transaction(
+            // $FlowFixMe
```

Comment:
Flow can't get the types of TransactionBuilder right. It complains saying `build()` not found in `TransactionBuilder`.  I suspect that when a type is used as input in a polymorfic type, in this case `BaseSignRequest<TransactionBuilder>`, for some reason Flow fails to check the types.

Reply:
Yeah, you have to use instanceof to cast it. I'd like to get rid of `BaseSignRequest` eventually since as you've probably notice, it doesn't work that well. The TxSignRequest makes a lot more logical sense (although it's questionable that it implements an interface and I may have to remove that some day)